spec/features/campagne_spec.rb

```diff
@@ -17,6 +17,9 @@
     before { visit admin_campagnes_path }
 
     it do
+      within '.panel_contents' do
+        expect(page).to_not have_content 'Compte'
+      end
```

Comment:
J'ai mis un certain temps à comprendre le lien entre le test et le code ajouté.

Je suggère de mettre cette vérification dans un test à part, peut-être de préciser un contexte à part (comme plus bas, `context 'en organisation' do… end`), peut-être enfin d'exprimer plus clairement l'intention du test (par exemple `it "ne permet pas de filtrer par compte" do… end`).

Par ailleurs, je pense qu'il serait bon d'ajouter un test qui vérifie qu'en tant qu'organisateur, on peut effectivement filtrer par compte.

Reply:
ok fait.